core/controlplane/config/templates/cluster.yaml

```diff
@@ -1319,6 +1322,8 @@ experimental:
       enabled: false
     OwnerReferencesPermissionEnforcement:
       enabled: false
+    # persistentVolumeClaimResize enables PersistentVolumeClaim in the API server, scheduler and kubelet. you need to manually add the
+    # featureGate ExpandPersistentVolumes:true in the worker
```

Comment:
@kiich @mumoshu, do you know when enabling the feature gate on node pools is necessary as per the comment? Is it only when control plane pods are scheduled onto workers?